merlin-server/constraints-dsl-compiler/src/libs/constraints-edsl-fluent-api.ts

```diff
@@ -956,6 +998,20 @@ declare global {
      * @param defaultProfile boolean or windows to take default values from
      */
     public assignGaps(defaultProfile: Windows | boolean): Windows;
+
+    /**
+     * Counts the cumulative amount of time spent in an active Span, as a multiple of some unit of time.
+     *
```

Comment:
Is this in the Windows class ? if yes, update comment from Span to Windows ?

Reply:
done